lms/djangoapps/discussion/rest_api/tests/test_views.py

```diff
@@ -461,12 +474,14 @@ def test_topic_id(self):
                             "id": "topic_id_1",
                             "thread_list_url": "http://testserver/api/discussion/v1/threads/?"
                                                "course_id=x%2Fy%2Fz&topic_id=topic_id_1",
-                            "name": "test_target_1"
+                            "name": "test_target_1",
+                            "thread_counts": {"discussion": 0, "question": 0},
                         }],
                         "id": None,
                         "thread_list_url": "http://testserver/api/discussion/v1/threads/?"
                                            "course_id=x%2Fy%2Fz&topic_id=topic_id_1",
-                        "name": "test_category_1"
+                        "name": "test_category_1",
+                        "thread_counts": None,
```

Comment:
Question: I am not able to understand this scenario. 

* Why `"thread_counts": None` here? 
* What is the difference between a category and a topic here? 
* Why we are associating **thread_counts** with a category that is only applicable to discussion topics?